MARC.js

```diff
@@ -574,8 +574,8 @@ record.prototype.translate = function(item) {
 		//  p = Name of part/section of a work
 		var titlesubfields = this.getFieldSubfields("245")[0];
 		item.title = glueTogether(
-			glueTogether(titlesubfields["a"], titlesubfields["b"], ": "),
-			glueTogether(titlesubfields["n"], titlesubfields["p"], ": "),
+			glueTogether(clean(titlesubfields["a"]), clean(titlesubfields["b"]), ": "),
+			glueTogether(clean(titlesubfields["n"]), clean(titlesubfields["p"]), ": "),
 			". "
 		);
 		item.title = clean(item.title);
```

Comment:
we can take out the clean here then, right?

Reply:
Maybe, but then I have to look closer what this function does besides deleting parenthesis...

Reply:
The only case where this would do something is where a parenthesis starts at the beginning of the title and ends at the end of the subtitle; that's illegal in MARC, right? If so, please remove the clean here.

Reply:
Yes, you are right. Moreover, there are only other cleaning steps at the beginning, the end, and for multiple spaces. Thus, we don't have to do these for the combination of the two strings, after we have cleaned them invidividually.